CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ about writing changes to this log.
 
 ## [Unreleased]
 
+## [2.1.0]
+
+- Drupal 10 compatibility.
```

Comment:
All (but one) other entries in the changelog start with a verb in the past tense …